sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -329,7 +335,36 @@ static struct snd_soc_codec_conf codec_conf[] = {
 
 };
 
-static struct snd_soc_dai_link dailink[] = {
+static struct snd_soc_dai_link_component rt1308_component[] = {
+	{
+		.name = "i2c-10EC1308:00",
+		.dai_name = "rt1308-aif",
+	}
+};
+
+static struct snd_soc_dai_link_component platform_component[] = {
+	{
+		/* name might be overridden during probe */
+		.name = "0000:00:1f.3"
+	}
+};
+
+struct codec_dai dai_lists[] = {
+	{
+		.id = 0x711,
+		.name = "rt711-aif1"
+	},
+	{
+		.id = 0x1308,
+		.name = "rt1308-aif"
+	},
+	{
+		.id = 0x715,
+		.name = "rt715-aif2"
+	},
+};
```

Comment:
can you add a comment on what this represents?

Reply:
yes, I will refine it. Thanks!